File: templates/js/sms.js
require('dotenv').config();
const accountSid = process.env.TWILIO_ACCOUNT_SID;
const authToken = process.env.TWILIO_AUTH_TOKEN;
const Twilio = require('twilio');
const client = new Twilio(accountSid, authToken);

client.messages
  .create({
     body: 'Hign risk of seizure predicted!',
     from: '+13133074053',
     to: '+18608997108'
   })
  .then(message => console.log(message.sid));

File: sms.py
import os
from dotenv import load_dotenv
from twilio.rest import Client

load_dotenv()

accountSid = os.getenv('TWILIO_ACCOUNT_SID')
authToken = os.getenv('TWILIO_AUTH_TOKEN')

client = Client(accountSid, authToken)

message = client.messages \
    .create(
         body='Sending from Python',
         from_= '+13133074053',
         to= '+18608997108'
     )

print(message.sid)
